### ISSUE TITLE
Export FormatType

### ISSUE THREAD

**Issue by beheh** [HearthSim/npm-deckstrings#3]
So that users of the library do not need to use the magical constants 1 and 2, but rather:

```typescript
import { encode, FormatType } from "deckstrings";

encode({
    ...params,
    format: FormatType.FT_WILD,
});
```

**Comment by Pyrolistical** [comment]
This has not been released yet

**Comment by beheh** [comment]
@Pyrolistical Thanks for the poke, I just released this as `deckstrings@2.1.0`.
